tox.ini

```diff
@@ -59,12 +73,29 @@ commands =
     python -c "import libtorrent as lt; print lt.__version__"
     python -m twisted.trial --reporter=deluge-reporter deluge.tests
 
+[testenv:wintrial]
+changedir = {toxinidir}
+setenv =
+    TOXENV=wintrial
+commands =
+    python -c "import libtorrent as lt; print lt.__version__"
+    python -m twisted.trial deluge.tests
+
 [testenv:plugins]
 setenv = PYTHONPATH = {env:PWD}:{env:PWD}/deluge/plugins
```

Comment:
here wont work in windows

Reply:
See fix here: https://github.com/deluge-torrent/deluge/pull/182/commits/3cbbd6186ee0b9fd58c6590917e3353623298299

Reply:
ok, ill re-run all the tests on windows and modify the commit as needed. ty.